build-setup/vscode/using_vscode.md

```diff
@@ -2,45 +2,45 @@
 
 Previously, we explained how to [set up CLion for `ml-cpp`](../clion/using_clion.md). You may find, that CLion has some
 performance problems when indexing `ml-cpp` project: it may take very long to open the project or CLion may freeze and
```

Comment:
I found CLion to be completely hopeless at reindexing the repo when we were using the old Makefiles solution.  I always had to kill it and regenerate `compile_commands.json` from scratch after a non-trivial change in the repo.

But with CMake it seems a lot better.  So maybe we could take out the bit about CLion freezing now.